nipype/pipeline/plugins/multiproc.py

```diff
@@ -352,6 +393,8 @@ def _send_procs_to_workers(self, updatehash=False, graph=None):
                 self._remove_node_dirs()
                 free_memory_gb += next_job_gb
                 free_processors += next_job_th
+                if is_gpu_node:
+                    free_gpu_slots -= next_job_gpu_th
```

Comment:
Note that this is releasing resource claims that were made around line 356 so the next time through the loop sees available resources.

```suggestion
                if is_gpu_node:
                    free_gpu_slots += next_job_gpu_th
```